workflow-ui/core-android/src/main/java/com/squareup/workflow1/visual/AndroidViewMultiRendering.kt

```diff
@@ -26,93 +22,47 @@ import com.squareup.workflow1.visual.ContextOrContainer.AndroidContext
  * to bind to the [AndroidViewMultiRendering] key.
  */
 @OptIn(WorkflowUiExperimentalApi::class)
-public class AndroidViewMultiRendering : MultiRendering<Context, View>() {
+public class AndroidViewMultiRendering : MultiRendering<ContextOrContainer, View>() {
 
   override fun create(
     rendering: Any,
-    context: Context,
+    context: ContextOrContainer,
```

Comment:
@wugeorgeq Stumbled across this when catching up to your work, missed it when reviewing earlier. FYI.